### DISCUSSION TITLE
How to customize the current active link?

### DISCUSSION THREAD

**Discussion by ataldev** [gilbitron/laravel-vue-pagination#133] [Help & support]
I want to remove the "current" word from the active pagination link. In the documentation, I could only find how to do it for the previous and next links.

**Comment by mrkazmierczak** [comment]
It's basic knowledge of CSS. At first of all you should add class to parent div of your navigation. So I added pagination.

`<div class="grid w-full place-items-center pagination"><TailwindPagination :data="items" :limit="2" @pagination-change-page="page => getItems(page)" class=""></TailwindPagination></div>`

And then override styles of this pagination like this for example:

`<style>
.pagination nav > button {
    border: none;
}
</style>`

You can also select the last and the first element like this, and change the content using CSS and remove (current). But I think this current is connected with aria attribute.

**Comment by DeveloperMohammedAmine** [comment]
the simple solution is :
<style>
    .sr-only{
        display: none !important;
    }
</style>

because "current" word on every click it append on the button . by this way you can make it indisplayed
